sklearn/impute.py

```diff
@@ -94,6 +113,13 @@ class SimpleImputer(BaseEstimator, TransformerMixin):
           each column.
         - If "most_frequent", then replace missing using the most frequent
           value along each column.
+        - If "constant", then replace missing values with fill_value.
+
```

Comment:
Please note here (or in a Notes section) that most_frequent and constant work with strings or numeric data, while the others only work with numeric data.